stubs/openpyxl/openpyxl/worksheet/dimensions.pyi

```diff
@@ -30,7 +30,7 @@ class Dimension(Strict, StyleableObject):
     outlineLevel: Integer[Literal[True]]
     outline_level: Alias
     collapsed: Bool[Literal[False]]
-    style: Alias
+    style: Alias  # type: ignore[assignment]
```

Comment:
```suggestion
    style: Alias
```
This change doesn't feel necessary. Like, it should already be failing if that was an issue?